### ISSUE TITLE
[Higher Level Review]: Some elements are not announcing correctly in JAWS

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#106044]
## What happened?

**Describe what went wrong**:

This bug is from a recent 508 Office accessibility audit on the Higher Level Review form. It's potentially across all our forms, not just one, but that requires more testing.

> Defect 4 | High | Created:
> WCAG 4.1.1 Parsing | In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features.
> 
> Throughout the product, when the screen is initially loaded, the names for the radio buttons are read once when reading with the arrow keys but after selecting a radio button, the names are read twice by assistive technology. For example, on the Request A Higher-Level Review | Veterans Affairs screen, when reading with the arrow keys, “Disability compensation claim radio button not checked” is read but after selecting this option, “Disability compensation claim radio button checked Disability compensation claim” is read with the arrow keys. (New: 03/19/2025)
> 
> Throughout the product, some elements are not read correctly by assistive technology. For example, on the Request A Higher-Level Review | Veterans Affairs screen (navigate to Step1 of 4Veteran information), under Personal information, when tab is used to move the focus to the Find more detailed instructions for how to … link, “Find more detailed instructions for how to change your legal name on file (opens in a new tab)opens in a new tab Link” is read by assistive technology—“opens in a new tab” is read twice. Similarly, on the Request A Higher-Level Review | Veterans Affairs screen (navigate to Step4 of 4Review application), “privacy policy (opens in a new tab)opens in a new tab Link” is read. (New: 03/19/2025)
> 
> Throughout the product, some elements are not read correctly by assistive technology. For example, on the Request A Higher-Level Review | Veterans Affairs screen (Step2 of 4Issues for review), when reading the check boxes with the arrow keys, the names are read twice—“The service connection check box not checked Disagreement with test decision on December 13, 2024 The service connection Disagreement with test decision on December 13, 2024” is read for the first check box. Additionally, when selecting the second instance of the name, “Request A Higher-Level Review | Veterans Affairs document 49 characters allowed” is read. (New: 03/19/2025)

**Add the steps to reproduce the bug**:

1. Go to: https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/ while logged in with a test user.
2. Turn on JAWS and navigate to the screens mentioned.
3. Note the way controls announce.

**Add a screen recording or photo**:

I plan to do more testing on these issues, and add more details.

## How severe is the problem?

- [ ] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [x] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [ ] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user encounters elements, the elements should announce properly in screen readers._

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place

**Comment by davidakennedy** [comment]
The part of the ticket that calls out links not announcing properly will be taken care of by a [ticket in the design system, related to the va-link component](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3699). It's being worked on.

**Comment by Mottie** [comment]
I went ahead and [fixed the `va-link` duplicate text in this PR](https://github.com/department-of-veterans-affairs/component-library/pull/1552)
